plugin-locationcomponent/src/test/java/com/mapbox/maps/plugin/locationcomponent/LocationPuckManagerTest.kt

```diff
@@ -257,12 +251,23 @@ class LocationPuckManagerTest {
     every { Value.fromJson(any()) } returns ExpectedFactory.createValue(Value("expression"))
     locationPuckManager.styleScaling(settings)
     verify { locationLayerRenderer.styleScaling(capture(valueSlot)) }
-    assertEquals(
-      "[interpolate, [exponential, 0.5], [zoom], 0.5, [literal, [$MODEL_SCALE_CONSTANT, $MODEL_SCALE_CONSTANT, $MODEL_SCALE_CONSTANT]], 22.0, [literal, [1.0, 1.0, 1.0]]]",
-      valueSlot.captured.toString()
-    )
+    val value = valueSlot.captured.toString()
```

Comment:
any comments what happens below?

Reply:
Well since we can't compare the whole string (it contains formatted double values that may be slightly different - we compare start and end of the strings, then parse double values and compare relative to ULP (smallest possible double value increment)